### DISCUSSION TITLE
diary/24oct2/

### DISCUSSION THREAD

**Discussion by giscus** [SunnyCloudYang/SunnyCloudYang.github.io#29] [Comments]
# diary/24oct2/

世界上只有一种长期主义，就是长期不改变，改变不长期

https://origakid.top/diary/24oct2/

**Comment by SunnyCloudYang** [comment]
后记：我发现那个UPS其实能用，前面是因为外壳挡住了供电的Type C所以一直接触不良，我的派根本食不饱力不足，所以才一直重启，现在真的可以撑住十个小时左右了🤦‍♂️
